Source/UI/Controllers/CardViewController.swift

```diff
@@ -332,4 +328,14 @@ public class CardViewController: UIViewController,
         validateFieldsValues()
     }
 
+    // MARK: Utility Methods
+    private func logPaymentFormPresented(isNextLogSuppressed: Bool,
+                                         checkoutApiClient: CheckoutAPIClient) {
+        if !isNextLogSuppressed {
+            checkoutApiClient.logger.add(metadata: checkoutApiClient.correlationID(),
+                                          forKey: .correlationID)
+            checkoutApiClient.logger.log(.paymentFormPresented)
+            suppressNextLog = true
```

Comment:
Now that this is more than just suppressing the next log, can we make this more descriptive? We're now logging once per correlation ID, would something like `loggedForCurrentCorrelationID` make more sense?